### PR TITLE
Feat: Added Login and Signup pages

### PR DESCRIPTION
**Pull request by LordUttam**
Added Login and signup pages. Please review thank you :).
image
image

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <link rel="stylesheet" href="../styles/auth.css">
+
+    <title>Padhaku | Login</title>
+</head>
+
+<body>
+    <header class="header--std">
+        <nav class="nav--std">
+            <ul class="header__list--align-start">
+                <!-- <li class="header__btn"><i class="fa fa-bars"></i></li> -->
```
Comment (chanchal16) [inline pages/login.html]: remove the commented code

File: pages/login.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                <li class="header__title"><a href="../index.html">Padhaku</a></li>
+            </ul>
+            <div class="search-container">
+                <div class="input__container flex--inline">
```
Comment (chanchal16) [inline pages/login.html:24]: be consistent with your class naming structure. Either use __ or --.

File: pages/signup.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    <link rel="stylesheet" href="../styles/auth.css">
+
+    <title>Padhaku | Register</title>
+</head>
+
+<body>
+    <header class="header--std">
+        <nav class="nav--std">
+            <ul class="header__list--align-start">
+                <!-- <li class="header__btn"><i class="fa fa-bars"></i></li> -->
```
Comment (bharati-21) [inline pages/signup.html]: You can remove the commented out code.

File: pages/login.html (hunk 1 above)
Comment (bharati-21) [inline pages/login.html]: You can remove the commented out code

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

File: styles/auth.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  font-size: 1rem;
+}
+
+p.signup-link,
+p.login-link {
+  color: var(--primary-color);
+  font-weight: bold;
+}
+
+.t-and-c {
```
Comment (chanchal16) [inline styles/auth.css]: what's t-and-c? try to give a more meaningful name.

File: styles/main.css [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+  width: 100%;
+}
+
+a {
+  display: flex;
+  justify-content: space-between;
+
+  width: 100%;
+  height: 100%;
+
```
Comment (chanchal16) [inline styles/main.css:15]: remove unnecessary blank spaces

File: pages/login.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (AtulHande03a) [inline pages/login.html:7]: add author meta tag

**Review by AtulHande03a — APPROVED** [review]

File: pages/login.html (hunk 2 above)
Comment (LordUttam) [reply pages/login.html:24]: Chanchal that is called BEM. __ signify sub elements and -- signify modifier.

File: pages/login.html (hunk 6 above)
Comment (LordUttam) [reply pages/login.html:7]: Done :) thank you.

File: styles/auth.css (hunk 4 above)
Comment (LordUttam) [reply styles/auth.css]: changed it to terms-conditions

File: pages/login.html (hunk 1 above)
Comment (LordUttam) [reply pages/login.html]: Done Bharati!

**Pull request merged**